src/engine/sys/sdl_glimp.cpp

```diff
@@ -1131,51 +1117,31 @@ static void GLimp_InitExtensions()
 		glConfig2.drawBuffersAvailable = true;
 	}
 
+	glConfig2.getProgramBinaryAvailable = false;
 #ifdef GL_ARB_get_program_binary
 	if( GLEW_ARB_get_program_binary )
 	{
 		int formats = 0;
 
 		glGetIntegerv( GL_NUM_PROGRAM_BINARY_FORMATS, &formats );
 
-		if ( !formats )
+		if ( formats == 0 )
 		{
-			logger.Notice("...GL_ARB_get_program_binary found, but with no binary formats");
-			glConfig2.getProgramBinaryAvailable = false;
-		}
-		else
-		{
-			logger.Notice("...using GL_ARB_get_program_binary");
-			glConfig2.getProgramBinaryAvailable = true;
+			logger.WithoutSuppression().Notice("...no program binary formats");
 		}
+
+		glConfig2.getProgramBinaryAvailable = LOAD_EXTENSION_WITH_TEST(ARB_get_program_binary, formats > 0);
 	}
-	else
+#else
+	NoticeExtUnsupported("ARB_get_program_binary");
 #endif
-	{
-		logger.Notice("...GL_ARB_get_program_binary not found");
-		glConfig2.getProgramBinaryAvailable = false;
-	}
 
+	glConfig2.bufferStorageAvailable = false;
 #ifdef GL_ARB_buffer_storage
```

Comment:
Can we just get rid of these ifdefs? I don't see a need for compiling against ancient versions of the GL headers.

Reply:
Maybe, I would make a separate commit so if some people with ancient software comes, he can revert or look at the code to know how to workaround.

Or, I may extend those macro to do the `#ifdef` check anytime (does preprocessor do multiple pass?) that would make the code ready for GL 4.7 extensions while avoiding failures when compiling Xenial. 🙈

Reply:
I removed those.